src/rgw/rgw_admin.cc

```diff
@@ -1,1011 +1,55 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
-#include <errno.h>
+#include <cerrno>
 #include <iostream>
 #include <sstream>
+#include <cstdlib>
```

Comment:
please use stdlib, but not sure why does you need to add this include

Reply:
Same as with cerrno vs errno.h